tests/trio200.py

```diff
@@ -39,3 +39,11 @@ async def bar():
             bar()  # TRIO200: 12, "bar", "BAR"
 
     bar()  # TRIO200: 4, "bar", "BAR"
+
+    # don't error on awaited expressions
+    # https://github.com/Zac-HD/flake8-trio/issues/85
+    await bar()
+    await print(bar())
```

Comment:
Ah, this actually *should* be an error, since you need to `print(await bar())`.  The tricky part is that we only want to skip the check if the _parent node_ of the call is an `await`, not _any ancestor_.  Consider e.g.

```python
await async_fn(
    value = blocking_fn()  # Want trio200: use `value = await nonblocking_fn()` here!
)
```

Reply:
Ah, right. Fixed :+1: